stanford_image.strongarm.inc

```diff
@@ -175,6 +175,18 @@ function stanford_image_strongarm() {
       'stanford_huge_landscape' => array(
         'custom_settings' => FALSE,
       ),
+      'stanford_banner_850x400' => array(
+        'custom_settings' => TRUE,
+      ),
+      'gallery_large' => array(
```

Comment:
@cjwest @boznik - These gallery strongarm settings cannot be in here. They belong in the stanford_gallery module. Please remove them.